errors.go

```diff
@@ -130,6 +130,18 @@ func (c cause) Error() string   { return c.Message() + ": " + c.Cause().Error()
 func (c cause) Cause() error    { return c.cause }
 func (c cause) Message() string { return c.message }
 
+// Errorf returns an error that formats as the given text.
```

Comment:
Errorf formats according to a format specifier and returns the string as a value that satisfies error.